inst/extdata/WQXDetectionQuantitationLimitTypeRef.csv

```diff
@@ -32,6 +33,7 @@
 "Detection/Quantitation Limit Type(DetectionQuantitationLimitTypeName)",17,"Statistical Uncertainty","That component of uncertainty which arises from imprecision. www.iaea.org/OurWork/ST/NE/NEFW/WTS-Networks/IDN/idnfiles/Wkp_on_Development_WM,WasteCharacterization_Clearance/Day3/12.Statistics_GREY.pdf","12/9/2015 7:55:38 AM","Other"
 "Detection/Quantitation Limit Type(DetectionQuantitationLimitTypeName)",18,"Systematic Uncertainty","That component of uncertainty which arises from biases. www.iaea.org/OurWork/ST/NE/NEFW/WTS-Networks/IDN/idnfiles/Wkp_on_Development_WM,WasteCharacterization_Clearance/Day3/12.Statistics_GREY.pdf","12/9/2015 7:55:38 AM","Other"
 "Detection/Quantitation Limit Type(DetectionQuantitationLimitTypeName)",28,"Taxonomic Loss Threshold","Taxonomic Loss Threshold","2/16/2023 12:05:38 PM","Other"
+"Detection/Quantitation Limit Type(DetectionQuantitationLimitTypeName)",37,"Upper 95% Confidence Limit","Upper limit of 95% confidence limit, the above point estimate within which the true value ","4/25/2023 2:58:03 PM","Non-Detect"
```

Comment:
Should this be referenced as an over-detect (above upper limit)?

Reply:
Good catch! - I will edit the GetDetLimitRef function

Reply:
After reviewing this again, I'm actually not sure if these detection limit values (95% confidence limit) should be assigned to the above or below detection groups. The definition looks incomplete, so it is not clear to me if these values correspond to the device/sample detection limits or not. Did you see these values used in an example dataset? If yes, could you share the dataset? @ehinman did you leave these out on purpose initially? Any thoughts?

Reply:
I don't think this issue existed before - when I was running through the checks for the histogram function, the tests failed bc the Detection Limit Reference Table needed to be updated (I think maybe it was updated by the WQX team recently? not sure..) so I ran the UpdateDetLimitRef function which added the new item for upper 95% confidence limit

Reply:
The domain table description is rather confusing.. "the above point estimate within which the true value" - is it missing some words at the end?

Reply:
Yes, nice to see the detection limit reference table test is working well! I agree both the upper and lower 95% fit best in the "Other" category

Reply:
Once you make this change, don't forget to run pkgdown::build_site() to update TADA GitHub pages site. After that, we should be ready to merge. Thanks again Katie!